__integration__/android.test.js

```diff
@@ -18,7 +18,7 @@ const {buildPath} = require('./_constants');
 describe('integration', () => {
   describe('android', () => {
     StyleDictionary.extend({
-      source: [`__integration__/tokens/**/*.json`],
+      source: [`__integration__/tokens/**/*.json?(c)`],
```

Comment:
This allows our token files to be `.json` or `.jsonc`. I made this change so I could add a comment in a token file about testing the use of leaving off `.value`.